src/framework/execution/executeIntegration.ts

```diff
@@ -69,7 +69,12 @@ async function executeIntegration(
 ): Promise<ExecuteIntegrationResult> {
   await removeStorageDirectory();
 
-  await config.validateInvocation?.(context);
+  try {
+    await config.validateInvocation?.(context);
+  } catch (err) {
+    context.logger.validationFailure(err);
+    throw err;
```

Comment:
Will this be caught and logged again? My hope is that it will not. In the production environment, it can be confusing to see the same thing logged multiple times in different formats (the Bunyan log output from `context.logger.validationFailure(err)` and the Lambda runtime logging of the same, uncaught error.

Reply:
Yeah, I can see multiple errors getting confusing and adding noise (our monitor has the same issue atm). I'm thinking we mark the errors that have been already logged and perform a check at the "catch-all" error handler to ensure duplicate logs are not written.

Reply:
There needs to be a way to handle gracefully handle execution failure and abort the sync/integration job anyway so I think I will handle this in the next pr.